src/components/popups/JobOffers.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import { useDispatch } from "@/hooks/useTypedDispatch";
+import { toggleBodyScrolling } from "@/store/feature/ui.slice";
+import Popup from "../ui/Popup";
+import Button from "../ui/button";
+import CloseIcon from "@/icons/close-icon.svg";
+import JobIcon from "@/icons/briefcase.svg";
+
+/**
+ * JobOffers interface props
+ *
+ * @interface JobOffersProps
+ * @typedef {JobOffersProps}
+ */
+interface JobOffersProps {
+  externalClick?: () => void;
+}
+
+/**
+ * JobOffers component
+ *
+ * @export
+ * @param {JobOffersProps} param0
+ * @param {() => void} param0.externalClick
+ * @returns {*}
+ */
+export default function JobOffers({ externalClick }: JobOffersProps) {
+  const [show, setShow] = useState(false);
+
+  const dispatch = useDispatch();
+
+  const onOpenJobOffers = () => {
+    setShow(!show);
+    dispatch(toggleBodyScrolling(true));
+  };
+
+  const onClose = () => {
+    externalClick && externalClick();
+    setShow(!show);
+    dispatch(toggleBodyScrolling(false));
+  };
+
+  return (
+    <>
+      <button className="hidden lg:inline-block underline text-primary bg-transparent pl-2 text-base capitalize" onClick={onOpenJobOffers}>
+        Job offers
+      </button>
+      <div className="flex lg:hidden">
+        <div className="w-10 h-10 ml-3 mr-2 my-3 rounded-full bg-blue-500">
+          <JobIcon className="m-2 text-white" />
+        </div>
+        <div className="py-5 font-medium text-gray-900 cursor-pointer" onClick={onOpenJobOffers}>
+          <p className="font-medium text-lg text-gray-900 capitalize">Job offers</p>
+        </div>
+      </div>
+      <Popup center show={show} className="px-3 pt-16 pb-2" overlayClassName="bg-white opacity-80" onClose={onClose}>
```

Comment:
Do we have a reason why we did not continue using the existing **classname** prop of the **popup component**.?

Reply:
styles in `className` is for the container of both the `overlay div` and the `children` in the `Popup`. I added `overlayClassName` to style the `div` used as the overlay of the `Popup`